src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Json/GraphQLNamedQueryRequestJsonConverter.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using GraphQL;
+
+namespace OrchardCore.Apis.GraphQL.Json;
+
+public class GraphQLNamedQueryRequestJsonConverter : JsonConverter<GraphQLNamedQueryRequest>
+{
+    public static readonly GraphQLNamedQueryRequestJsonConverter Instance = new();
+    
+    /// <summary>
+    /// Name for the operation name parameter.
+    /// See https://github.com/graphql/graphql-over-http/blob/master/spec/GraphQLOverHTTP.md#request-parameters
+    /// </summary>
+    private const string OperationNameKey = "operationName";
```

Comment:
private variable should start with `_` followed by lower-case.

```suggestion
    private const string _operationNameKey = "operationName";
```